### PR TITLE
Mession pull request

### PR DESCRIPTION
**Pull request by codehass**
Hi @MohamedHNoor 

## In this pull request : 
- Add mission component.
- style mission component.
- Add profile component.
- style profile component.
- configure the  store

Thank you for reviewing my PR 😎

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,19 +1,23 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (MohamedHNoor) [inline src/App.js:1]: Hi Hassan Please remove the eslint and disable the next line

Comment (MohamedHNoor) [inline src/App.js:1]: Hi @codehass  please remove the eslint-disable-next-line and fix the eslint error

**Review by MohamedHNoor — APPROVED** [review]
# Approved 🎉 
Hi @codehass  👋👋
Congratulations you did great work, your project is complete! There is nothing else to say other than... it's time to merge it 🚢 :
Congratulations! 👏 

## Highlights

- Gitflow best practices have been followed✔️
- Good commit history ✔️
- Linters are passing ✔️
- Descriptive PR 

### Suggestions
- Please remove the eslint disable that you applied and fix the error that was needed to disable eslint

File: src/App.js (hunk 1 above)
Comment (codehass) [reply src/App.js:1]: Thank you @MohamedHNoor I will make the changes 😎

**Review by MohamedHNoor — APPROVED** [review]
# Approved 🎉 
Hi @codehass  👋👋
Congratulations you did great work, your project is complete! There is nothing else to say other than... it's time to merge it 🚢 :
Congratulations! 👏 

## Highlights

- Gitflow best practices have been followed✔️
- Good commit history ✔️
- Linters are passing ✔️
- Descriptive PR

**Pull request merged**